.travis.yml

```diff
@@ -54,7 +53,7 @@ matrix:
   - rust: 1.32.0
     env:
     - LABEL="msrv"
-    script:
+re    script:
```

Comment:
Is this `re` here a typo or is it some fancy YAML? 🙂

Reply:
um... I am drunk... musta have typed in the wrong window